blogs/nlp/LOREN/README.md

```diff
@@ -0,0 +1,144 @@
+
+The above aggregation of the soft logic serves to distill the knowldge to $p_{\theta}(y|z,x)$ (Probability of the claim label given the phrase-level label and input x),
+which is the probability we eventually want to optimize.
+
+## Experiment
+
+The model is trained on BERT and RoBERTa, and eventually it is evaluated on the dataset FEVER 1.0 shared task that is split into training, development and blind test set.
+Before we analyze the experiment result, I will summarize the experiment metrics they used to evaluate the performance[2]:
+
+### Experiment metrics for the performance of the model:
+
+1. Label Accuracy (LA): The accuracy of predicted label for claim regardless of retrieved evidence.
+2. FEVER score (FEV): The accuracy of both predicted label and retrieved evidence.
+
+<p align="center"><img src="./result1.png" alt="image4"/></p>
+
+Let's look at the above overall performance of LOREN. We can see from the result that LOREN is either outperforming or comparable with other
+baseline method except for LisT5. This is mainly because the PLM for LisT5, which is T5, is larger than BERT and RoBERTa. However, LOREN still outperforms LisT5 on the FEVER score of the development set (dev). To summarize, the performance of LOREN is decent and it has the potential to achieve a better result.
+
+Here is a bar chart that can visualize LOREN's comparable test FEV score with other models:
+
+<p align="center"><img src="./bar_plot.png" alt="image4"/></p>
+
+<p align="center"><img src="./result2.png" alt="image4"/></p>
+
+In addition, we will dive into the most important part of the experiment: a case study of the LOREN's interpretability.
+For claim one in the above figure, LOREN can detect which phrase is the culprit in the claim, and it's also providing the reasoning that why that phrase is causing the mistake, i.e., replacing the false phrase "the number three" with "the number one". In the third claim, LOREN shows its ability to detect multiple culprit phrases in the claim. However, for the second claim, the predicted result should be NEI. LOREN made this mistake mainly because of the evidence retrieval module and MRC module. First, the evidence retrieval module did not draw the correct evidence, and then the MRC module still replaced the original phrase even though there's no direct evidence given. Here's some examples that indicates a drawback of LOREN:
+
+<p align="center"><img src="./result3.png" alt="image4"/></p>
+
+For the phrase "a tree" in the claim, even though LOREN generates the correct local premise based on the evidence, it still considers the label of this phrase as supported. It is because LOREN does not possess commonsense knowledge, and we can see the probability of SUP and REF are very close, indicating that LOREN fails to distinguish "tree" and "edible fruit produced" as two completely different objects.
+
+<p align="center"><img src="./claim1.png" alt="image4"/><img src="./claim2.png" alt="image4"/></p>
```

Comment:
Could you find better examples? Good examples can be reasonable and perhaps interesting/trending sentences that are easily verifiable by human. We will see how LOREN performs. We can group them into Good cases and failure cases. 

e.g. San Francisco is the largest city in California. 
The headquarter of Apple Inc is in California. 
The area of California is larger than Texas. 
Barack Obama's wife is Michelle Obama. 

Please use your imagination to create multiple diverse instances of claims and analyze the reason why LOREN succeeds or fails. This would be more interesting than the techniques.

Reply:
I added the example and I think I find the reason why LOREN's lack of common sense knowledge: It's about LOREN's evidence ranking. Sometimes LOREN does now contain the evidence of some phrases even though that evidence is very important for LOREN to understand that claim.